### PR TITLE
docs(guide/di): clarify example description

### PR DESCRIPTION
**Pull request by kambleaa007**
docs(IoC): modified description about example

**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

**What is the current behavior? (You can also link to an open issue here)**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?**

**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our [guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] Fix/Feature: [Docs](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#documentation) have been added/updated
- [ ] Fix/Feature: Tests have been added; existing tests pass

**Other information**:

### CODE REVIEW

File: docs/content/guide/di.ngdoc [hunk 1]
```diff
@@ -283,9 +283,12 @@ Here is an example of using the injector service:
 // Provide the wiring information in a module
 var myModule = angular.module('myModule', []);
 ```
+Notice that `myModule` is independent as array is empty here.
```
Comment (petebacondarwin) [inline docs/content/guide/di.ngdoc]:
```suggestion
Notice that `myModule`  depends upon no other modules as the second parameter is an empty array. 
```

File: docs/content/guide/di.ngdoc [hunk 2]
```diff
@@ -283,9 +283,12 @@ Here is an example of using the injector service:
 // Provide the wiring information in a module
 var myModule = angular.module('myModule', []);
 ```
+Notice that `myModule` is independent as array is empty here. 
+Dependent is always need dependency object to be injected inside it.
```
Comment (petebacondarwin) [inline docs/content/guide/di.ngdoc]: I don't understand this sentence. Remember that the previous code snippet is defining a module so it is not helpful to refer to "objects" here.

File: docs/content/guide/di.ngdoc [hunk 3]
```diff
@@ -283,9 +283,12 @@ Here is an example of using the injector service:
...
 
 Teach the injector how to build a `greeter` service. Notice that `greeter` is dependent on the
 `$window` service. The `greeter` service is an object that contains a `greet` method.
+so, injector now injects `$window` dependency inside `greeter` service.
```
Comment (petebacondarwin) [inline docs/content/guide/di.ngdoc]: I don't feel that this sentence makes sense. If anything it should appear before the previous sentence and should be something more like

> Notice that `greeter` is dependent on the `$window` service, which will be provided by the injector when it is instantiated.

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

**Review by kambleaa007 — COMMENTED** [review]
changed

**Comment by kambleaa007** [comment]
@petebacondarwin thanks,
Can you guide me how to make successful commit.

File: docs/content/guide/di.ngdoc [hunk 4]
```diff
@@ -283,9 +284,12 @@ Here is an example of using the injector service:
 // Provide the wiring information in a module
 var myModule = angular.module('myModule', []);
 ```
+Notice that `myModule`  depends upon no other modules as the second parameter is an empty array.
```
Comment (mgol) [inline docs/content/guide/di.ngdoc]: @petebacondarwin A double space got inserted here by accident:
```suggestion
Notice that `myModule` depends upon no other modules as the second parameter is an empty array. 
```

File: docs/content/guide/di.ngdoc [hunk 5]
```diff
@@ -14,7 +14,8 @@ and providing them to other components as requested.
 
 ## Using Dependency Injection
 
-Dependency Injection is pervasive throughout AngularJS. You can use it when defining components
+Dependency Injection is pervasive throughout AngularJS. 
+You can use it when defining components
```
Comment (mgol) [inline docs/content/guide/di.ngdoc]: Is this change necessary?

**Review by kambleaa007 — COMMENTED** [review]
Helpful for beginners

**Review by kambleaa007 — COMMENTED** [review]
Im fine with this Thanks

**Comment by kambleaa007** [comment]
Hello all,
I think i'm gettin errors as i dont have write access to repo.
Does anybody help me in this??

**Comment by petebacondarwin** [comment]
@kambleaa007 what errors? You should not need access to write to the main repository. You just write to your own fork,

**Comment by kambleaa007** [comment]
> @kambleaa007 what errors? You should not need access to write to the main repository. You just write to your own fork,

 Travis Giving me:- CLAs are signed, but unable to verify author consent

**Comment by petebacondarwin** [comment]
This is because @mgol and I have made commits to this PR. You can ignore this error.

**Comment by kambleaa007** [comment]
> This is because @mgol and I have made commits to this PR. You can ignore this error.

That's great,
This is going to be interesting, as i learned a lot, this would be my first commit if it succeed hopefully.

**Review by gkalpak — APPROVED** [review]

**Comment by gkalpak** [comment]
I tweaked this a bit and merged. Thx, @kambleaa007 👍

**Pull request closed without merging**